### DISCUSSION TITLE
%EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EB%AF%B8%EC%85%988/

### DISCUSSION THREAD

**Discussion by giscus** [SungbinYang/SungbinYang.github.io#11] [Comments]
# %EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EB%AF%B8%EC%85%988/

양성빈의 블로그

https://sungbin.kr/%EC%84%B1%EB%B9%88-%ED%81%B4%EB%9F%BD-%EB%AF%B8%EC%85%988/

**Comment by tdronnie** [comment]
배열은 메모리 할당이 연속적으로 되어서 인덱스 접근이나 캐싱에 있어서도 도움이 되겠군요... 정리 잘보았습니다!!

**Reply by SungbinYang** [reply]
감사합니다!
